platforms/hyperledger-indy/charts/indy-auth-job/templates/job.yaml

```diff
@@ -12,6 +12,7 @@ spec:
       restartPolicy: OnFailure
       imagePullSecrets:
         - name: "{{ $.Values.image.pullSecret }}"
+      when: (network.docker.username | default('', true) | trim != '') && (network.docker.password | default('', true) | trim != '')  
```

Comment:
This would not work for helmcharts. You can run `helm lint` to check

Reply:
Any suggested changes?

Reply:
you can check other helm charts how if condition is added.